.github/workflows/ci.yml

```diff
@@ -69,14 +69,14 @@ jobs:
         run: echo "cache=$(go env GOCACHE)" >> $GITHUB_OUTPUT
 
       - name: Cache the Go Build Cache
-        uses: actions/cache@6849a6489940f00c2f30c0fb92c6274307ccb58a # v4.1.2
+        uses: actions/cache@v4.2.3
```

Comment:
We should use commit hashes here rather than tagged versions, especially after the whole tjactions incident.

Reply:
Done!